text_collector_examples/smartmon.sh

```diff
@@ -184,8 +184,9 @@ for device in ${device_list}; do
   sat+megaraid*) /usr/sbin/smartctl -A -d "${type}" "${disk}" | parse_smartctl_attributes "${disk}" "${type}" ;;
   scsi) /usr/sbin/smartctl -A -d "${type}" "${disk}" | parse_smartctl_scsi_attributes "${disk}" "${type}" ;;
   megaraid*) /usr/sbin/smartctl -A -d "${type}" "${disk}" | parse_smartctl_scsi_attributes "${disk}" "${type}" ;;
+  nvme*) /usr/sbin/smartctl -A -d "${type}" "${disk}" | parse_smartctl_scsi_attributes "${disk}" "${type}" ;;
   *)
-    echo "disk type is not sat, scsi or megaraid but ${type}"
+      (>&2 echo "disk type is not sat, scsi, nvme or megaraid but ${type}")
```

Comment:
Why the subshell? For a single echo command, I would do `echo "..." >&2`.

Reply:
To avoid side effects. The file descriptor `2` might have been changed before this code gets to run. Pretty sure this is a none issue at the moment, but who knows what the future might bring. Imho its just good practice if one wants to be sure to print to `stderr`.

Reply:
Ahh, yes, good point. Thanks.